### ISSUE TITLE
replace `dist_spec()` code with `get_parameter()` + `Gamma()` or `LogNormal()`

### ISSUE THREAD

**Issue by avallecam** [epiverse-trace/tutorials-early#2]
replace 

https://github.com/epiverse-trace/tutorials-early/blob/8dcbabf3577f754eba1961a4a09a9bd6e64ab9b0/episodes/read-delays.Rmd#L814-L822

from

```r
# update: EpiNow2 1.4.9 ---------------------------------------------------

si_x <- seq(1L, to = covid_serialint_discrete_max, by = 1L)

serial_interval_covid_update <- 
  EpiNow2::dist_spec(pmf = covid_serialint_discrete$prob_dist$d(si_x))

serial_interval_covid_update
#> - nonparametric distribution
#>   PMF: [0.0073 0.1 0.2 0.19 0.15 0.11 0.075 0.051 0.034 0.023 0.016 0.011 0.0076 0.0053 0.0037 0.0027 0.0019 0.0014 0.001 0.00074 0.00055 0.00041 0.00031]

```

~key learning goals~
- ~a simple way to plug in epiparameter is using the `pmf`~
- ~we can also use functions like `LogNormal()` or `Gamma()` but these require more (unnecessary) parameters~
- ~we need `max` parameter to plug in `LogNormal()` output to `epinow()`~
- ~the `max` parameter can come from a discretized continuous function as an `epidist` object~

**Comment by avallecam** [comment]
change my mind. the simpler input is still using the explicit distribution parameters

```r
# new branch: is this a simpler and valid alternative? --------------------------------

covid_serialint_parameters <- epiparameter::get_parameters(covid_serialint)

covid_serialint_parameters
#>   meanlog     sdlog 
#> 1.3862617 0.5679803

serial_interval_covid_branch_lognormalmax <- 
  EpiNow2::LogNormal(
    meanlog = covid_serialint_parameters["meanlog"],
    sdlog = covid_serialint_parameters["sdlog"],
    max = covid_serialint_discrete_max)

serial_interval_covid_branch_lognormalmax
#> - lognormal distribution (max: 23):
#>   meanlog:
#>     1.4
#>   sdlog:
#>     0.57
```

epidist
- get_parameters()
- discrete
  - max

compared to

epidist
- discrete
- max
- seq
- density
- pmf

**Comment by avallecam** [comment]
this update depends on the merging of https://github.com/epiforecasts/EpiNow2/pull/504

**Comment by avallecam** [comment]
fixed in #80
